src/modules/balance/ModuleTokensValue.vue

```diff
@@ -43,7 +43,7 @@
         <div class="mew-heading-3 mb-3 black--text">
           Total Value: ${{ totalTokenValues }}
         </div>
-        <module-tokens class="pa-0" dense />
+        <module-tokens class="pa-0" dense @trade="showPopup = false" />
```

Comment:
Just use the `handleTokensPopup` already defined as a prop for `<module-tokens>`

Reply:
fixed